doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,143 @@
+| `req.cim.04` | General | The Architecture **must** include image repository management. |
+| `req.cim.05` | General | The Architecture **should** support a multi-tenanted environment. |
+| `req.cim.06` | General | The Architecture **must** support resource tagging. |
+| `req.cim.07` | General | The Architecture **must** support horizontal scaling. |
+| `req.cim.08` | General | The Architecture **must** support workload resiliency. |
+
+<p align="center"><b>Table 2-3:</b> Containerised Platform: CIM Requirements.</p>
+
+<a name="2.3.4"></a>
+### 2.3.4 Interfaces & APIs
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.int.api.01` | API | The Architecture **must** provide Control API endpoints to cloud platform core services. |
+| `req.int.api.02` | API | The Architecture **must** provide APIs needed to discover and manage NFVI resources. |
+
+
+<p align="center"><b>Table 2-4:</b> Containerised Platform: Interfaces and APIs Requirements. </p>
+
+<a name="2.3.5"></a>
+### 2.3.5 Operations and LCM of the Infrastructure
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.lcm.gen.01`	| General | The Architecture **must** support zero downtime expansion/change of physical capacity (compute hosts, storage increase/replacement). |
+| `req.lcm.adp.01` | Automated deployment | The Architecture **should** allow for “cookie cutter” automated deployment, configuration, provisioning and management of multiple NFVI sites. |
+| `req.lcm.adp.02` | Automated deployment | The Architecture **must** support hitless upgrades of software provided by the cloud provider so that the availability of running workloads is not impacted. |
+| `req.lcm.adp.03` | Automated deployment | The Architecture **should** support hitless upgrade of all software provided by the cloud provider that are not covered by `req.lcm.adp.02`. Whenever hitless upgrades are not feasible, attempt should be made to minimize the duration and nature of impact. |
+| `req.lcm.adp.04` | Automated deployment | The Architecture **must** support declarative specifications of hardware and software assets for automated deployment, configuration, maintenance and management. |
+| `req.lcm.adp.05` | Automated deployment | The Architecture **should** support automated process for Deployment and life-cycle management of CIM Instances. |
+| `req.lcm.cid.02` | CI/CD | The Architecture **should** support integration with CI/CD Toolchain for NFVI and CIM components Automation. |
+
+<p align="center"><b>Table 2-5:</b> Containerised Platform: LCM Requirements. </p>
```

Comment:
Suggest this is changed to "Kubernetes Architecture: LCM Requirements"

Reply:
Changed to: "Kubernetes Architecture: Operations and LCM Requirements"